### PR TITLE
Changelog for v3.8.5

### PR DESCRIPTION
**Pull request by BoD**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+# Version 3.8.5
+
+_2024-07-10_
+
+Version 3.8.5 is a maintenance release. New developments happen in the 4.x versions.
+
+* [all] Bump okio to 3.9.0 (#5868)
+* [runtime] Adding checks for json end_document in http transport (#5894)
+* [runtime] Adding checks for json end_document in http batching interceptors (#5892)
+* [codegen] Fix mapping of the builtin Float scalar type (#6047)
```
Comment (martinbonnin) [inline CHANGELOG.md:13]: I think there is https://github.com/apollographql/apollo-kotlin/pull/5975 as well?

Comment (BoD) [reply CHANGELOG.md:13]: true! 👀

Comment (martinbonnin) [reply CHANGELOG.md]: Wait, sorry I meant #5972 I don't think we usually include docs changes in the changelog

Comment (BoD) [reply CHANGELOG.md]: 🤦 Right!

**Review by martinbonnin — APPROVED** [review]

**Pull request merged**